Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -319,7 +320,7 @@
   - BARTENDERS
   - BOTANISTS
   - CAPTAINS
-  - CAPTAINS AND HEADS
+  - CAPTAINS AND HEADS OF STAFF
```

Comment:
"Heads" is an awkward shorthand name for the heads of staff so just for clarity.